web/source/keymanweb.js

```diff
@@ -3525,22 +3535,106 @@
     // IE: call _SelectionChange when the user changes the selection 
     if(document.selection  &&  !keymanweb.legacy)
       util.attachDOMEvent(document, 'selectionchange', keymanweb._SelectionChange);
-       
-    // Add event listeners and attach manually-positioned KMW objects
-    if(keymanweb.options['attachType'] != 'manual')  // I1961
```

Comment:
Does the mutation observer code work with `keymanweb.options['attachType'] = 'manual'`? That is, only attach to controls that the page JS tells KMW to?